HW7/phone_book.py

```diff
@@ -0,0 +1,65 @@
+phone_book = {}
+
+# Додавання контакту
+def add_contact():
+    name = input("Введіть ім'я: ")
+    phone_number = input("Введіть номер телефону: ")
+    phone_book[name] = phone_number
+    print("Контакт додано.")
+
+# Видалення контакту
+def delete_contact():
+    name = input("Введіть ім'я контакту, який хочте видалити: ")
+    if name in phone_book:
+        del phone_book[name]
+        print("Контакт видалено.")
+    else:
+        print("Контакт не знайдено.")
+
+# Виведення списку всіх контактів
+def list_contacts():
+    if not phone_book:
+        print("Телефонна книга порожня.")
+    else:
+        print("Контакти:")
+        for name, phone_number in phone_book.items():
+            print(f"{name}: {phone_number}")
+
+# Пошук контакту за ім'ям та виведення його детальної інформації
+def show_contact():
+    name = input("Введіть ім'я контакту, який хочете переглянути: ")
+    if name in phone_book:
+        print(f"Ім'я: {name}")
+        print(f"Номер телефону: {phone_book[name]}")
+    else:
+        print("Контакт не знайдено.")
+
+# Кількість записів
+def show_stats():
+    print(f"У телефонній книзі {len(phone_book)} запис(ів).")
+
+# Команди
+while True:
+    print("Щоб додати конттакт, введіть 'додати'.")
+    print("Щоб видалити контакт, введіть 'видалити'.")
+    print("Щоб вивести список контактів, введіть 'список'.")
+    print("Щоб переглянути контакт, введіть 'переглянути'.")
+    print("Щоб вийти з програми, введіть 'вийти'.")
+    print("Щоб переглянути кількість контактів у книзі, введіть 'кількість'.")
+
+    command = input("Введіть команду: ").lower()
+
+    if command == "додати":
```

Comment:
nit: better to use English words in code in order to other programmers from any country could understand what is going on there